app/uk/gov/hmrc/tariffclassificationfrontend/controllers/AuthenticatedAction.scala

```diff
@@ -52,10 +53,15 @@ class AuthenticatedAction @Inject()(appConfig: AppConfig,
       Some(request.session)
     )
 
-    authorise().retrieve(Retrievals.credentials and Retrievals.name) {
-      case ~(credentials: Credentials, name: Name) =>
+    authorised((Enrolment(teamEnrolment) or Enrolment(managerEnrolment)) and AuthProviders(PrivilegedApplication)).retrieve(Retrievals.credentials and Retrievals.name and Retrievals.allEnrolments) {
+      case (credentials: Credentials) ~ (name: Name) ~ (roles: Enrolments) =>
         val id = credentials.providerId
-        block(AuthenticatedRequest(Operator(id, name.name), request))
+        val operator = Operator(
+          id,
+          name.name,
+          manager = roles.enrolments.map(_.key).contains(managerEnrolment)
```

Comment:
Why operator does not save all the enrolments and the manager is part of of out business logic (ex.: hasManagerRole)?

Reply:
Thats what this `manager` flag is @pllaurado